pkg/volume/glusterfs/glusterfs.go

```diff
@@ -438,6 +577,24 @@ func (d *glusterfsVolumeDeleter) Delete() error {
 
 	glog.V(4).Infof("glusterfs: deleting volume %q with configuration %+v", volumeId, d.provisioningConfig)
 
+	gid, err := d.getGid()
+	if err != nil && err != ErrNotFound {
+		glog.Errorf("glusterfs: failed to parse the gid")
+		return fmt.Errorf("glusterfs: failed to parse the gid")
```

Comment:
ok. probably true. delete nonetheless and possibly sacrifice a gid? next restart it'll be fixed